### PR TITLE
Added search icon

### LINKED ISSUE #111: Adding a Search Icon in the Search bar
**Is your feature request related to a problem? Please describe.**
No, my feature request is not related to any problem. It's about making the page more attractive.

**Describe the solution you'd like**
I want to add a "Search Icon" in the "Search Bar" to give a better look to the project.

**Are you willing to contribute to this issue?** [Yes/No]
Yes

**Describe alternatives you've considered**
As of now, I have no alternatives for this, I just want to add a "Search Icon" in the "Search Bar".

**Additional context**
Screenshot (2706)_LI

@maneprajakta I want to contribute this as a part of GSSOC'21.Please assign this issue to me.

### PR DESCRIPTION
**Pull request by Sweeti-Gupta**
Fixed #111

## Description
Search icon has been added in the search bar.
#111 

## Affected Dependencies
NA

## How has this been tested?
Using: python manage.py runserver

## GIF Before
Kitabe_before

## GIF After
Kitabe_after

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/Praful932/Kitabe/blob/master/CONTRIBUTING.md).
- [x] My changes do not edit/add any unrequired files.
- [x] My changes are covered by tests.

Hey @maneprajakta , I have added the search icon and fixed issue #111 .Please review and merge it.

### CODE REVIEW

**Review by maneprajakta — CHANGES_REQUESTED** [review]
[Unstage](https://stackoverflow.com/questions/38743912/remove-a-file-from-a-git-pull-request) `settings.py`

File: templates/mainapp/layout.html [hunk 1]
```diff
@@ -15,91 +14,92 @@
     <link href="https://fonts.googleapis.com/css2?family=Fredoka+One&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Kalam&display=swap" rel="stylesheet">
     <!-- Bootstrap CSS -->
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
```
Comment (Praful932) [inline templates/mainapp/layout.html:18]: Please don't edit lines that need not be changed

**Review by Praful932 — CHANGES_REQUESTED** [review]
@Sweeti-Gupta There are too many edits for just adding the search icon, please reset your commit and do only the necessary changes

File: templates/mainapp/layout.html (hunk 1 above)
Comment (Sweeti-Gupta) [reply templates/mainapp/layout.html:18]: Ok Sir!

**Comment by Sweeti-Gupta** [comment]
@Praful932 , @maneprajakta , I have done all the necessary changes in the code for adding the search icon. Please review and merge it.

**Comment by maneprajakta** [comment]
@Sweeti-Gupta there are conflicts.
Also, the commits are coming from @MrDevLovePer ?

**Comment by Sweeti-Gupta** [comment]
I am new to open source and doesn't know much about github and merge
conflicts. What happened is I mistakenly deleted the cloned folder and then
cloned it again and make a new branch to push changes. Now it is showing
conflict.Please help me in solving the conflict.
The commits are from @MrDeveloper, because I am using his system to commit
changes as I was facing issues in setting up the project in my system. He
is my friend.

**Comment by Praful932** [comment]
@Sweeti-Gupta No problem, It's fine :) 
If you prefer, you can open a fresh PR, otherwise
Just reset the commits back to master in your branch and then do fresh changes and force push

**Comment by Sweeti-Gupta** [comment]
I will try it tomorrow. Thank you for the help.

**Comment by Sweeti-Gupta** [comment]
@Praful932 Do I need to close this PR for creating a fresh PR? What will be the steps for creating a fresh PR? Do I need to clone it again and create a new branch and then start working on that branch?Please guide me.

**Comment by Praful932** [comment]
> @Praful932 Do I need to close this PR for creating a fresh PR? What will be the steps for creating a fresh PR? Do I need to clone it again and create a new branch and then start working on that branch?Please guide me.

Yes you are correct, close the PR and follow the steps in the [guidelines](https://github.com/Praful932/Kitabe/discussions/42) :)

**Pull request closed without merging**